2008-07-25  Magnus Granberg  <zorry@ume.nu>

	* include/libiberty.h (asprintf): Don't declare if defined as a macro

--- include/libiberty.h.zorry
+++ include/libiberty.h
@@ -554,8 +554,11 @@
 /* Like sprintf but provides a pointer to malloc'd storage, which must
    be freed by the caller.  */
 
+/* asprintf may be declared as a macro by glibc with __USE_FORTIFY_LEVEL.  */
+#ifndef asprintf
 extern int asprintf (char **, const char *, ...) ATTRIBUTE_PRINTF_2;
 #endif
+#endif
 
 #if !HAVE_DECL_VASPRINTF
 /* Like vsprintf but provides a pointer to malloc'd storage, which
